src/App.jsx

```diff
@@ -14,70 +15,55 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [curGood, setCurGood] = useState('Jam');
 
```

Comment:
please pay attention to naming...what is `cur` means?

Reply:
current )